modules/core/src/main/java/org/apache/ignite/internal/processors/platform/cache/query/PlatformFieldsQueryCursor.java

```diff
@@ -70,6 +84,17 @@ public PlatformFieldsQueryCursor(PlatformContext platformCtx, QueryCursorEx<List
             for (int i = 0; i < cnt; i++) {
                 writer.writeString(fq.getFieldName(i));
             }
+        } else if (type == OP_GET_FIELDS_META) {
+            QueryCursorEx<List<?>> cursor = cursor();
+
+            List<GridQueryFieldMetadata> metadatas = cursor.fieldsMeta();
```

Comment:
`metadatas` -> `metas`

Reply:
https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules

Weird rules, but we follow them.